ui/components/app/transaction-alerts/transaction-alerts.js

```diff
@@ -27,11 +48,41 @@ const TransactionAlerts = ({
   const pendingTransactions = useSelector(submittedPendingTransactionsSelector);
   const t = useI18nContext();
 
+  ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+  const trackEvent = useContext(MetaMetricsContext);
+  const txParams = txData?.txParams || {};
+  const methodData = useSelector((state) =>
+    getKnownMethodData(state, txParams?.data),
+  );
+  ///: END:ONLY_INCLUDE_IN
+
+  ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+  const onClickSupportLink = useCallback(() => {
+    trackEvent({
+      category: MetaMetricsEventCategory.Transactions,
+      event: MetaMetricsEventName.ExternalLinkClicked,
+      properties: {
+        action: 'Confirm Screen',
+        legacy_event: true,
+        recipientKnown: null,
+        functionType:
+          'confirm' ||
+          getMethodName(methodData.name) ||
+          TransactionType.contractInteraction,
```

Comment:
I'm a bit confused by this. wouldn't 
```
'confirm' ||
          getMethodName(methodData.name) ||
          TransactionType.contractInteraction
```
always resolve to 'confirm'?

Reply:
Resolved from removing `functionType`